src/components/common/input/index.tsx

```diff
@@ -0,0 +1,18 @@
+import React, { FC } from "react";
+import styles from "./Input.module.css";
+
+type InputProps = {
+  label: string;
+  inputType: string;
+};
+
+const Input: FC<InputProps> = ({ label, inputType }) => {
+  return (
```

Comment:
Same comment here for the prop name and the implicit return

Reply:
Also, `inputType` could be required or use a default value

Reply:
Great, I have change the implicit return & prop name, also added a default value. Thanks